contracts/src/traits/access_control/mod.rs

```diff
@@ -18,7 +18,7 @@
 // LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 // OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-
+#[allow(clippy::module_inception)]
```

Comment:
what is it?

Reply:
clippy is complaining about module inception. I think we should keep it as it is, but stop clippy from complaining about it 😃